core/interfaces/status.py

```diff
@@ -386,11 +464,30 @@ class GsStatusOpenFileCommand(TextCommand, GitCommand):
     """
 
     def run(self, edit):
-        lines = util.view.get_lines_from_regions(self.view, self.view.sel())
-        file_paths = (line.strip() for line in lines if line[:4] == "    ")
```

Comment:
Here, we had to `strip` and check for `    ` 4 leading spaces because we had no fine grained selectors. This is all over the place herein.